i18n/translations/en.json

```diff
@@ -413,16 +413,15 @@
 
 	"license": "License",
 	"license.activate": "Activate it now",
+	"license.activate.label": "Please activate your site",
+	"license.activate.domain": "Your license will be activated for <strong>{host}</strong>.",
+	"license.activate.local": "You are about to use your Kirby license to activate your local domain <strong>{host}</strong>. If this site will be deployed to a public domain, please activate it there instead. If {host} is the domain you want to register with your license, please continue.",
 	"license.buy": "Buy a license",
-	"license.register": "Register",
+	"license.code.help": "You received your license code after the purchase via email. Please copy and paste it to activate your site.",
+	"license.code.label": "Please enter your license code",
 	"license.manage": "Manage your licenses",
 	"license.ready": "Ready to launch your site?",
-	"license.register.help": "You received your license code after the purchase via email. Please copy and paste it to register.",
-	"license.register.label": "Please enter your license code",
-	"license.register.domain": "Your license will be registered to <strong>{host}</strong>.",
-	"license.register.local": "You are about to register your license for your local domain <strong>{host}</strong>. If this site will be deployed to a public domain, please register it there instead. If {host} is the domain you want to license Kirby to, please continue.",
-	"license.register.success": "Thank you for supporting Kirby",
-	"license.unregistered": "This is an unregistered demo of Kirby",
+	"license.success": "Thank you for supporting Kirby",
 	"license.unregistered.label": "Unregistered",
```

Comment:
Suggestion for an alternate term for the stats card: "Trial mode"

Reply:
I could see that - but would be good to get more opinions here. Shall we merge this as is and tackle this when adding also the v4 license wording in (after implementing the backend handling)?

Reply:
Either way is fine for me. The other changes in this PR are already a huge improvement.

Reply:
I like Trial mode. But the wording could also raise questions if there are some deactivated features. Let's merge this already and think about it some more in the next steps as Nico suggested.